simulator/src/main/java/com/hazelcast/simulator/worker/loadsupport/StreamerFactory.java

```diff
@@ -60,13 +60,27 @@ private StreamerFactory() {
         return new SyncMapStreamer<K, V>(map);
     }
 
+    public static <K, V> Streamer<K, V> getInstance(IMap<K, V> map, int concurrencyLevel) {
```

Comment:
same here - you do not want to have the same code duplicated. why the other `getInstance()` does not simply delegate to this method?